src/content/en/2020/security.md

```diff
@@ -729,15 +726,15 @@ Some other security mechanisms are even required to prevent certain types of att
 
 The [`X-XSS-Protection`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection) header, which is currently adopted by 18.39% of the websites, was used to control the browser's built-in detection mechanism for reflected cross-site scripting. However, as of Chrome version 78, the built-in XSS detector has been [deprecated and removed](https://bugs.chromium.org/p/chromium/issues/detail?id=968591) from the browser. This was because there exist various bypasses, and the mechanism also introduced new [vulnerabilities and information leaks](https://frederik-braun.com/xssauditor-bad.html) that could be abused by attackers. As the other browser vendors never implemented a similar mechanism, the `X-XSS-Protection` header effectively has no effect on modern browsers and can thus safely be removed. Nevertheless, we do see a slight increase in the adoption of this header compared to last year, from 15.50% to 18.39%.
 
 The remainder of the top five most widely adopted headers is completed by two headers related to a website's TLS implementation. The [`Strict-Transport-Security`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) header is used to instruct the browser that the website should only be visited over an HTTPS connection for the duration defined in the `max-age` attribute. We explored the configuration of this header in more detail [earlier in this chapter](#transport-security). The `Expect-CT` header will instruct the browser to verify that any certificate that is issued for the current website needs to appear in public [Certificate Transparency](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT) logs.
 
-Overall, we can see that the adoption of security headers has increased in the last year: the most-widely used security headers show a relative increase of 15 to 35 percent. Interestingly, also the adoption of the features that were introduced more recently, such as the `Report-To` and `Feature-Policy` headers, is ramping up. The features are adopted by almost twice as many sites compared to last year. The strongest growth can be seen for the CSP header, with an adoption rate growing from 4.94% to 10.93%.
+Overall, we can see that the adoption of security headers has increased in the last year: the most-widely used security headers show a relative increase of 15 to 35 percent. The growth adoption of the features that were introduced more recently, such as the `Report-To` and `Feature-Policy` headers, is also worth noting—the latter has more than tripled compared to last year. The strongest absolute growth can be seen for the CSP header, with an adoption rate growing from 4.94% to 10.93%.
```

Comment:
"The growth adoption" feels like it's missing an "in" or "of"

Reply:
```suggestion
Overall, we can see that the adoption of security headers has increased in the last year: the most-widely used security headers show a relative increase of 15 to 35 percent. The growth in adoption of the features that were introduced more recently, such as the `Report-To` and `Feature-Policy` headers, is also worth noting—the latter has more than tripled compared to last year. The strongest absolute growth can be seen for the CSP header, with an adoption rate growing from 4.94% to 10.93%.
```